cmd/image-builder-db-test/main_test.go

```diff
@@ -109,6 +109,26 @@ func insertData2(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func insertData3(t *testing.T) {
+	conn := connect(t)
+	defer conn.Close(context.Background())
+
+	insert := "INSERT INTO composes(job_id, request, created_at, account_number, org_id) VALUES ($1, $2, CURRENT_TIMESTAMP - interval '2 days', $3, $4)"
+
+	_, err := conn.Exec(context.Background(), insert, uuid.New().String(), "{}", ANR3, ORGID1)
+	require.NoError(t, err)
+
+	insert = "INSERT INTO composes(job_id, request, created_at, account_number, org_id) VALUES ($1, $2, CURRENT_TIMESTAMP - interval '3 days', $3, $4)"
+
+	_, err = conn.Exec(context.Background(), insert, uuid.New().String(), "{}", ANR3, ORGID1)
+	require.NoError(t, err)
+
+	insert = "INSERT INTO composes(job_id, request, created_at, account_number, org_id) VALUES ($1, $2, CURRENT_TIMESTAMP - interval '4 days', $3, $4)"
+
+	_, err = conn.Exec(context.Background(), insert, uuid.New().String(), "{}", ANR3, ORGID1)
+	require.NoError(t, err)
+}
+
```

Comment:
Since this operates on data that existed since the first migration, and it's not specific to the 3rd migration file, this should be moved to insertData1. I added the comment on line 79 for this but could also rename the functions insertDataMigration1/2/3/...